app/views/unattended/provisioning_templates/snippet/remote_execution_ssh_keys.erb

```diff
@@ -33,7 +36,7 @@ if [ -z "$PKG_MANAGER" ]; then
 <%= indent(2) { snippet 'pkg_manager' } -%>
 fi
 
-<% if !host_param('remote_execution_ssh_keys').blank? %>
+<% if host_param('remote_execution_ssh_keys').present? || host_param('remote_execution_ssh_ca_keys').present? %>
```

Comment:
This looks suspicious. Previously we did something when A was blank and now we do it when A is NOT blank or B is NOT blank?